apps/mobile/src/components/TradeChecklistFlow/components/AgreeOnTradeDetailsScreen/index.tsx

```diff
@@ -9,10 +18,31 @@ function AgreeOnTradeDetailsScreen({
     params: {chatId, inboxKey},
   },
 }: Props): JSX.Element {
+  const {t} = useTranslation()
+  const goBack = useSafeGoBack()
+  const tradeChecklistUuid = useAtomValue(tradeChecklistUuidAtom)
+  const offerForTradeChecklist = useAtomValue(offerForTradeChecklistAtom)
+  const createTradeCheckListStateFromChat = useSetAtom(
+    createTradeCheckListStateFromChatActionAtom
+  )
+
+  useEffect(() => {
+    createTradeCheckListStateFromChat({chatId, inboxKey})
+  }, [chatId, createTradeCheckListStateFromChat, inboxKey, tradeChecklistUuid])
```

Comment:
We can place `tradeChecklistUuid` into params and make the interface for the rest of the app more clear